client/gutenberg/editor/style.scss

```diff
@@ -1,3 +1,22 @@
+// Block library
+@import '../../../node_modules/@wordpress/block-library/build-style/style';
```

Comment:
You can convert all these imports to:
```
@import '@wordpress/block-library/build-style/style';
```
now. Unlike the legacy pipeling, the imports are processed by webpack and module resolution works the same ways as for JS modules.

I want to verify a few more details before approving, but I don't expect any issues. Thank you for fixing!

Reply:
That causes me the following error:

```
ERROR in ./client/gutenberg/editor/style.scss
Module build failed (from ./node_modules/mini-css-extract-plugin-with-rtl/dist/loader.js):
ModuleBuildError: Module build failed (from ./node_modules/sass-loader/lib/loader.js):
File to import not found or unreadable: @wordpress/block-library/build-style/style.
in /Users/mmtr/dev/wp-calypso/client/gutenberg/editor/style.scss (line 3, column 1)
```

I can solve it by using `@import '~@wordpress/block-library/build-style/style';` instead

Reply:
> I can solve it by using @import '~@wordpress/block-library/build-style/style'; instead

Yes that would be great. I didn't know that `sass-loader` requires the `~` prefix to do a module import.